srml/babe/src/lib.rs

```diff
@@ -318,6 +327,15 @@ impl<T: Trait> Module<T> {
 	}
 
 	fn do_initialize() {
+		// since do_initialize can be called twice (if session module is present)
+		// => let's ensure that we only modify the storage once per block by checking
```

Comment:
Out-of-date comment

Reply:
Why do you think so?

Reply:
OK - I see that "by checking block number" is obsolete :)